src/ui/ErrorReport/ErrorReport.ts

```diff
@@ -187,21 +211,42 @@ export class ErrorReport extends Component {
   }
 
   private buildErrorInfo(data: IEndpointError): HTMLElement {
-    let errorInfo = $$('div', {
+    const errorInfo = $$('div', {
       className: 'coveo-error-info'
     });
 
     let textArea = $$('textarea', undefined, JSON.stringify(data, null, 2));
     errorInfo.el.appendChild(textArea.el);
 
-    let infoLabel = $$('div', {
+    const infoLabel = $$('div', {
       className: 'coveo-error-info-label'
     }, l('CopyPasteToSupport'));
     errorInfo.el.appendChild(infoLabel.el);
 
     return errorInfo.el;
   }
 
+  private buildOptionsElement() {
+    const optionsElement = $$('div', { className: 'coveo-error-report-options' });
+    optionsElement.el.appendChild(this.buildPrevious());
+    optionsElement.el.appendChild(this.buildReset());
+    optionsElement.el.appendChild(this.buildRetry());
+    this.container.append(optionsElement.el);
+  }
+
+  private buildEndpointErrorElements(helpLink?: string) {
+    this.helpSuggestion.empty();
+    const defaultHelpLink = 'http://www.coveo.com/go?dest=cloudhelp&lcid=9&context=254';
```

Comment:
You can simply put a default value for the parameter 

`buildEndpointErrorElements(helpLink: string = http://www.coveo.com/go?dest=cloudhelp&lcid=9&context=254)`